django/chat/templates/chat/chat.html

```diff
@@ -45,7 +45,7 @@
           <i class="bi bi-gear"></i>
           <span class="visually-hidden">{% trans "Toggle chat options sidebar" %}</span>
         </button>
-        <div class="container px-5">
+        <div class="container px-5 position-relative">
```

Comment:
The position-relative class allows us to use sticky positioning on the scroll down button.